### ISSUE TITLE
The CI environment variable is not set in GitHub Actions

### ISSUE THREAD

**Issue by eregon** [actions/virtual-environments#520]
**Describe the bug**

The CI environment variable is not set in GitHub Actions. A lot of software relies on that to detect whether it's running in CI.
Example: https://github.com/jruby/jruby/issues/6113#issuecomment-596196601

**Area for Triage**: 

**Question, Bug, or Feature?**: 
Feature

**Virtual environments affected**
- [x] macOS 10.15
- [x] Ubuntu 16.04 LTS
- [x] Ubuntu 18.04 LTS
- [x] Windows Server 2016 R2
- [x] Windows Server 2019

**Expected behavior**
The environment variable CI should be set, as it is in most CI solutions out there (e.g., TravisCI).

**Actual behavior**
It's not set: https://github.com/ruby/ruby/runs/493161205

**Comment by al-cheb** [comment]
Hello, @eregon 
I have prepared https://github.com/actions/virtual-environments/pull/530 PR to the CI variable.

**Comment by maxim-lobanov** [comment]
@al-cheb , I have a concern. I am not sure that this variable should be really set during image-generation. What if customer builds his own vhd based on our source code, and run it locally? In this case, this variable will be set but actually, it won't real CI.

In my opinion, this environment variable should be set by actions/runner on build queue.

@al-cheb , @alepauly , any thoughts?

**Comment by alepauly** [comment]
> 
> In my opinion, this environment variable should be set by actions/runner on build queue.
> 
> @al-cheb , @alepauly , any thoughts?

@maxim-lobanov I agree. Transferring to actions/runner where if fixed then it will work for self-hosted runners as well.
